pocs/state/states/default/tracking.py

```diff
@@ -1,14 +1,31 @@
+from pocs.utils import current_time
+
+
 def on_enter(event_data):
     """ The unit is tracking the target. Proceed to observations. """
     pocs = event_data.model
-    pocs.say("Checking our tracking")
+    pocs.next_state = 'parking'
+
+    # If we came from pointing then don't try to adjust
+    if event_data.transition.source != 'pointing':
+        pocs.say("Checking our tracking")
+        try:
+            ra_info, dec_info = pocs.observatory.update_tracking()
```

Comment:
Seems like we should move more of this code into POCS or Observatory. i.e. these on_enter functions should have little logic of their own. Thoughts?

Reply:
Yes, good point. Doing that now.

Reply:
I've done this but removed the `say` output that says what correction was. It's been useful for me the last few days to watch it but it's also a little too much information for the `say` box. Ideally we should be plotting this in real-time within PAWS. The info is delivered via the `status` update so information is there.

This was mostly because I've been keeping any `say` calls exclusively in the states (except for the setup in `core.py`). Not sure if that is necessary or not.